src/data/full-quiz.js

```diff
@@ -1,4 +1,106 @@
+    Distractor2: "getYear()",
+    Distractor3: "getFullYear()",
+    Explanation: 
+    "The getFullYear() method returns the year of the specified date according to local time. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear",
+  },
+
+{
+    Question:
+     "Which of the following method returns the day of the hour for the specified date, according to local time?",
+    Answer: "getHours()",
+    Distractor1: "getHours()",
+    Distractor2: "getMilliseconds()",
+    Distractor3: "getMinutes()",
+    Explanation: 
+    "The getHours() method returns the hour for the specified date, according to local time. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getHours",
+  },
+{
+    Question:
+     "Which of the following method returns the milliseconds in the specified date according to local time?",
+    Answer: "getMilliseconds()",
+    Distractor1: "getHours()",
+    Distractor2: "getMilliseconds()",
+    Distractor3: "getMinutes()",
+    Explanation: 
+    "The getMilliseconds() method returns the milliseconds in the specified date according to local time. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMilliseconds",
+  },
+{
+    Question: 
+    "Which of the following method returns the minutes in the specified date according to local time?",
+    Answer: "getMinutes()",
+    Distractor1: "getHours()",
+    Distractor2: "getMilliseconds()",
+    Distractor3: "getMinutes()",
+    Explanation: 
+    "The getMinutes() method returns the minutes in the specified date according to local time. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMinutes",
+  },
+{
+    Question: 
+    "Which of the following method returns the month in the specified date according to local time, as a zero-based value?",
+    Answer: "getMonth()",
+    Distractor1: "getMonth()",
+    Distractor2: "getSeconds()",
+    Distractor3: "getTime()",
+    Explanation:
+     "The getMonth() method returns the month in the specified date according to local time, as a zero-based value (where zero indicates the first month of the year). ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth",
+  },
+{
+    Question:
+     "Which of the following method returns the seconds in the specified date according to local time?",
+    Answer: "getSeconds()",
+    Distractor1: "getMonth()",
+    Distractor2: "getSeconds()",
+    Distractor3: "getTime()",
+    Explanation: 
+    "The getSeconds() method returns the seconds in the specified date according to local time. ",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getSeconds",
+  },
+{
+    Question: 
+    "Which of the following method returns returns the number of milliseconds* since the Unix Epoch?",

```

Comment:
method should be methods.
You wrote returns twice.
Also, please remove the * since that was supposed to denote further explanation in the docs.

Proposed fix: 
Which of the following methods returns the number of milliseconds since the Unix Epoch?